### ISSUE TITLE
docker-compose up fails for Routerlicious server

### ISSUE THREAD

**Issue by Flavyoo** [microsoft/FluidFramework#4903]
## Describe the bug
I'm trying to spin up the routerlicious server on my local machine, but running `docker-compose up` results in an authorization error.

```
Pulling gateway (prague.azurecr.io/gateway:latest)...
ERROR: Head https://prague.azurecr.io/v2/gateway/manifests/latest: unauthorized: authentication required, visit https://aka.ms/acr/authorization for more information.
```

### To Reproduce
1. Clone the FluidFramework Repo
2. Cd into **FluidFramework/server/routerlicious**
3. run `docker-compose build`
4. run `docker-compose up`
5. See error

### Expected behavior

The Routerlicious service runs on port 3000.

**Comment by curtisman** [comment]
Yep.  the docker-compose needs to update to the public docker feed.

**Comment by Flavyoo** [comment]
@curtisman if the public feed is available, I can update the docker-compose files to get it working.

**Comment by curtisman** [comment]
@Flavyoo, yes.  `server/docker-compose.yml` is already updated.

The one in server/routerlicious remap local build to the container for dev inner loop to test local changes.  Since most people would have access to our private feed, we haven't notice that it hasn't updated yet.

**Comment by Flavyoo** [comment]
Okay I will update it. Can you assign this bug to me?

**Comment by Flavyoo** [comment]
@curtisman I checked your changes [here](https://github.com/microsoft/FluidFramework/commit/37bc8ba817da7daa3b740690e2b32ee63db6c3ab#diff-384e92019121a5f7537238f61bc89717e9a9bde9af213a6a8aea7a703f3c9136L3) and saw you removed the gateway, auspkn, and verdaccio services from the docker-compose.

However, `gateway` and `auspkn` were the services that could not be built because of the authentication error. Can you confirm if these services should be in the `server/routerlicious` docker-compose file or not.

**Comment by curtisman** [comment]
@Flavyoo.  Yes, those can be removed.   Previously, gateway/auspkn are the front end and package service we used for testing.  
Now we have the `webpack-dev-server` plug-in in the client packages that allow it to target the docker image, so for routerlicious usage and development, we don't need to run those by default any more.  

You should remove verdaccio as well.

Thanks in advance in doing the change.
